lib/get_started/get_started_view.dart

```diff
@@ -0,0 +1,121 @@
+              child: Image.asset(
+                'assets/images/strawberry2-removebg-preview.png',
+                height: screenHeight * 0.07,
+                width: screenWidth * 0.13,
+              ),
+            ),
+            Padding(
+              padding: EdgeInsets.symmetric(horizontal: screenWidth * 0.04),
+              child: Column(
+                children: [
+                  const Spacer(flex: 4),
+                   Align(
+                    alignment: Alignment.topLeft,
+                    child: Padding(
+                      padding: const EdgeInsets.only(left: 20.0),
+                      child: Text(
+                        'Lorem ipsum dolor sit amet,\nconsectetur adipiscing elit sed.',
+                        textScaler: TextScaler.linear(ScaleSize.textScaleFactor(context)),
+                        style:  TextStyle(
+                            color: Colors.brown,
+                            fontSize: ScreenUtil().setSp(16),
+                            fontWeight: FontWeight.bold),
+                      ),
+                    ),
+                  ),
+                  const SizedBox(height: 20),
+                   Align(
+                    alignment: Alignment.topLeft,
+                    child: Padding(
+                      padding: EdgeInsets.only(left: 20.0),
+                      child: Text(
+                        'Tempor incididunt ut labore et \ndolore magna aliqua consectetur \nadipiscing elit sed.',
+                        style: TextStyle(
+                          color: Colors.brown,
+                          fontSize: ScreenUtil().setSp(16),
+                        ),
+                        textScaler: TextScaler.linear(ScaleSize.textScaleFactor(context)),
+                      ),
+                    ),
+                  ),
+                  const Spacer(flex: 2),
+                  Align(
+                    alignment: Alignment.center,
+                    child: ElevatedButton(
+                      onPressed: () {},
+                      style: ElevatedButton.styleFrom(
+                        padding: const EdgeInsets.symmetric(vertical: 16.0, horizontal: 40.0),
+                        foregroundColor: const Color(0xFFf3e1d4),
+                        backgroundColor: const Color(0xFFf3e1d4),
+                        shape: RoundedRectangleBorder(
+                          borderRadius: BorderRadius.circular(30),
+                        ),
+                      ),
+                      child: Text(
+                        'Get Started',
+                        textScaler: TextScaler.linear(ScaleSize.textScaleFactor(context)),
+                        style: TextStyle(color: Colors.brown,fontSize:ScreenUtil().setSp(16)),
+                      ),
+                    ),
+                  ),
+                  SizedBox(height: screenHeight * 0.05),
+                ],
+              ),
+            ),
+          ],
+        ),
+      ),
+    );
+  }
+}
```

Comment:
![image](https://github.com/user-attachments/assets/45a279cd-075d-4972-ae43-1005097a08e0)
Image going throw system bar

Reply:
Andestrial fix this issue and made it flexible